.github/workflows/test-get-vault-secrets.yaml

```diff
@@ -28,6 +29,12 @@ jobs:
           - invalid
     runs-on: ubuntu-latest
 
+    # The `get-vault-secrets` action only works when run from a `grafana`
+    # repository, so skip this test if the PR is from a different repository. We
+    # will still get a run of this workflow for the change before merging, as we
+    # use merge queues.
```

Comment:
I'm assuming here that a merge queue branch will be in the target repository (and so `get-vault-secrets` will work) but I don't actually know this. We'll find out when we try, if the CI still runs and passes 🙃.